src/test/suite/explorer/explorerController.test.ts

```diff
@@ -204,7 +152,7 @@ suite('Explorer Controller Test Suite', function () {
           `Expected active connection to be 'localhost:27018' found ${instanceId}`
         );
 
-        // This will timeout in 1s, which is enough time for us to just check.
+        // This will timeout in 500ms, which is enough time for us to just check.
```

Comment:
From where we got this 500ms value?

Reply:
The driver url we try to connect on is `mongodb://shouldfail?connectTimeoutMS=1000&serverSelectionTimeoutMS=500`
Which will fail/timeout in 500ms.

Reply:
Got ya!